lib/experimental/fonts/font-library/class-wp-font-utils.php

```diff
@@ -133,5 +133,74 @@ function ( $elem ) {
 
 			return join( ';', $slug_elements );
 		}
+
+		/**
+		 * Sanitize a tree of data using an schema that defines the sanitization to apply to each key.
+		 *
+		 * It removes the keys not in the schema and applies the sanitizator to the values.
+		 *
+		 * @since 6.5.0
```

Comment:
Do you think we should mark this one as `@access private` like the others in this class, in case we want to change how it works in the future?

Reply:
it makes sense. added in https://github.com/WordPress/gutenberg/pull/58571/commits/78a0b848b437cdd7ccde0534ed4406afe1683f57